src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,122 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEAFULT_SIZE = 16;
+    static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    int size;
+
+    private Entry<K, V>[] table;
```

Comment:
The array 'table' should use the diamond operator to ensure type safety. Change the declaration to 'new Entry<K, V>[DEFAULT_SIZE]' and suppress the unchecked assignment warning, as Java does not allow the direct creation of generic arrays.